### PR TITLE
updates to minikube build instructions

### PR DESCRIPTION
**Pull request by rshriram**

### CODE REVIEW

**Comment by kyessenov** [comment]
Test failed with `W0815 14:35:40.585] [2017-08-15T14:32:41.703Z] "GET /b HTTP/1.1" 431 - 0 0 256 255 "-" "Go-http-client/1.1" "bca2b860-c938-9e1f-9626-532836faeffa" "b" "10.20.3.87:80"`

**Comment by kyessenov** [comment]
Caused by #978 for b to b HTTP request.

File: doc/minikube.md [hunk 1]
```diff
@@ -28,15 +28,16 @@ manifests in inability to pull images.
...
 
 3. Run bazel build and test
 
        kubectl exec bazel -it bash
        cd /pilot
 
        # make sure platform/kube/config exists; touch an empty file to use the minikube cluster for testing
-       # sudo touch platform/kube/config
+       # Symlinks to ${HOME}/.kube/config will not work.
+       sudo touch platform/kube/config
```
Comment (ldemailly) [inline doc/minikube.md]: why sudo ?

Comment (rshriram) [inline doc/minikube.md]: not sure if that works with rest of the flow.. if it does, it can be default

**Pull request merged**